src/main/java/com/synopsys/integration/alert/channel/jira/cloud/web/JiraCloudCustomEndpoint.java

```diff
@@ -73,25 +72,24 @@ public JiraCloudCustomEndpoint(AuthorizationManager authorizationManager, JiraCl
         try {
             JiraCloudServiceFactory jiraServicesCloudFactory = jiraProperties.createJiraServicesCloudFactory(logger, gson);
             PluginManagerService jiraAppService = jiraServicesCloudFactory.createPluginManagerService();
-            String username = jiraProperties.getUsername();
-            String accessToken = jiraProperties.getAccessToken();
-            Response response = jiraAppService.installMarketplaceCloudApp(JiraConstants.JIRA_APP_KEY, username, accessToken);
-            if (BooleanUtils.isTrue(response.isStatusCodeError())) {
-                return new ActionResponse<>(HttpStatus.BAD_REQUEST, "The Jira Cloud server responded with error code: " + response.getStatusCode());
+            int statusCode = jiraAppService.installMarketplaceCloudApp(JiraConstants.JIRA_APP_KEY);
+            if (!HttpStatusCodes.isSuccess(statusCode)) {
+                return new ActionResponse<>(HttpStatus.BAD_REQUEST, "The Jira Cloud server responded with error code: " + statusCode);
             }
 
-            boolean jiraPluginInstalled = JiraPluginCheckUtil.checkIsAppInstalledAndRetryIfNecessary(jiraAppService, username, accessToken);
+            boolean jiraPluginInstalled = JiraPluginCheckUtil.checkIsAppInstalledAndRetryIfNecessary(jiraAppService);
             if (!jiraPluginInstalled) {
-                return new ActionResponse<>(HttpStatus.NOT_FOUND, "Unable to confirm Jira Cloud successfully installed the Jira Cloud plugin. Please verify the installation on you Jira Cloud server.");
+                return new ActionResponse<>(HttpStatus.NOT_FOUND, String.format(
+                    "Unable to confirm successful installation of the Jira Cloud '%s' plugin. Please verify the installation on you Jira Cloud server.", JiraConstants.JIRA_ALERT_APP_NAME));
```

Comment:
Existing issue but "Please verify the installation on **you** Jira Cloud server."  Should change to "Please verify the installation on **your** Jira Cloud server."